ext/devcontainer/src/radcli/README.md

```diff
@@ -0,0 +1,29 @@
+# Radius CLI
+
+Installs the [Radius CLI](https://github.com/radius-project/radius) along with needed dependencies.
+
+## Example Usage - Install latest release
+
+```json
+"features": {
+    "ghcr.io/devcontainers/radius/radiuscli:latest": {
+        "version": "latest"
+    }
+}
+```
+
```

Comment:
## Example Usage - Install specific release

This will install the specified release of the `rad` CLI. In this example the release version is `0.28.0`.

```json
"features": {
    "ghcr.io/devcontainers/radius/radiuscli:latest": {
        "version": "0.28.0"
    }
}
```